packages/mirinae/src/inputs/dropdown/select-dropdown/PSelectDropdown.vue

```diff
@@ -244,6 +246,17 @@ watch(() => props.disabled, (disabled) => {
 
     throw new Error('If \'multiSelectable\' is \'true\', \'selected\' option must be an array.');
 })();
+(async () => {
+    if (props.initSelectedWithHandler && props.handler && !props.disableHandler) {
+        // this is to refine selected items by handler's results whose label is fully set.
+        const { results } = await props.handler('', undefined, undefined, state.proxySelectedItem);
+        state.proxySelectedItem = state.proxySelectedItem.map((item) => {
+            const found = results.find((d) => d.name === item.name);
+            if (found) return found;
+            return item;
+        });
+    }
```

Comment:
Suggestion: It would be better to extract this code into a function like 'initForFilterableDropdown' (for example)!